### ISSUE TITLE
Issues in the configuration

### ISSUE THREAD

**Issue by ecstrema** [TerosTechnology/vscode-terosHDL#680]
Here an aglomerate of issues related to the settings UI and implementation.

1) You currently have an array of tools, but only one is selected in the General page. All others should not be shown, and only the configuration for this tool should show up, in the same page, just below the tool selection under a section like "Tool options". 

<details>

![image](https://github.com/user-attachments/assets/6a439224-9b2b-4559-85f6-1f7e0fa25c61)
![image](https://github.com/user-attachments/assets/87e7093c-2824-4dfc-aea9-6236b1fcba3a)

</details>

2) The last point also applies to the linter and formatter settings.
3) A lot of these options seem completely unrelated: 
	- GTKWave installation directory should probably be in a GTWWave tool page
	- The waveform viewer selection should come before all gtkwave options
	- The manual compilation order setting is among those, but has nothing to do. Plus, it seems like something that's more related to a specific tool, although I don't know what it does, and couldn't find documentation for it.
	- And that's just for 1 page with 6 fields.
4) All sections wich have a single subsection should be a single page. You should not have to first expand the section, then click on "General"

<details>

This one is actually pretty funny:
![image](https://github.com/user-attachments/assets/ff824d05-fc74-4c55-9d29-e7c8b8e94d07)

</details>

Actually, all of this is mostly trivial things, but since the documentation is very very sparse, and examples even rarer, it makes up for something that's super hard to work with, although once you get the hang of it, it works incredibly well!

My recommandation may be very radical, but that is how other extensions work, and I think it works very well.

1) Move all settings related to the user (not to the project) to vscode's own settings mechanism.
	1) All binary paths (python, make, gtkwave, etc)
	2) All preferences (stutter mode, schematic viewer, etc)
2) Move all tool settings to configuration files. This is often how these tools are configured anyway.
3) Is there anything remaining?

Overall, thank you for TerosHDL, it works super well, and integrates nicely with VSCode.

Have a good day!

**Comment by jevogel** [comment]
I agree, it would be a lot easier to share a development environment with colleagues if we could use the standard VS Code configuration files. Then we could check them into our Git repos like we do with our other VS Code configurations. This would be even easier than the "Export" and then "Import" flow that is currently required.

**Comment by qarlosalberto** [comment]
I did some improvements in the next release.
